airflow/providers/cncf/kubernetes/operators/custom_object_launcher.py

```diff
@@ -59,9 +59,9 @@ def validate(self):
         if self.spec.get("dynamicAllocation", {}).get("enabled"):
             if not all(
                 [
-                    self.spec["dynamicAllocation"]["initialExecutors"],
-                    self.spec["dynamicAllocation"]["minExecutors"],
-                    self.spec["dynamicAllocation"]["maxExecutors"],
+                    self.spec["dynamicAllocation"].get("initialExecutors"),
+                    self.spec["dynamicAllocation"].get("minExecutors"),
+                    self.spec["dynamicAllocation"].get("maxExecutors"),
```

Comment:
Looks like the changes not only about add unit tests

Reply:
yes also added a minor fix
by not using `get` to pick out any keys under dynamicAllocations, the thrown error looks confused with `KeyError` exception instead of the ` AirflowException`, which should be the expected one

Reply:
should we edit the PR title instead?

Reply:
Yeah, just reflect it into the PR description title. Because otherwise it might accidentally skipped into the Changelog during the release process.

Reply:
thanks, updated